ern-api-gen/resources/swift/libraries/ern/api.mustache

```diff
@@ -67,3 +137,4 @@ public class {{classname}}Requests: NSObject {
     {{/operations}}
 }
 
+#endif
```

Comment:
Looking at this file, it seems like the only difference between swift3/swift4 is about prefixing the class declaration with `@objcMembers`. The rest of the code is exactly the same (might be wrong just had a quick look).
If that's the case no need to duplicate the whole source, as its harder to see where are the difference between swift3/4 in the file. It also complexifies maintenance as everything is duplicated.
What I would do, given the minimal differences is just surround the classes declarations with the preprocessor directive, i.e :

```
#if swift(>=4.0)
@objcMembers public class {{classname}}: NSObject  {
#else
public class {{classname}}: NSObject  {
#endif
```